fastlane/Fastfile

```diff
@@ -240,3 +248,7 @@ def replace_in(previous_text, new_text, path)
   backup_extension = '.bck'
   sh("sed", '-i', backup_extension, sed_regex, path)
 end
+
+def current_version_number
+  File.read("../.version")
```

Comment:
yes, I love this. I was thinking about doing this for the `latest` stuff but ended up not doing it. I like it

Reply:
definitely makes things easier